bin/sbank

```diff
@@ -461,10 +460,13 @@ else:
 opts_dict = dict((k, v) for k, v in opts.__dict__.items()
                  if v is not False and v is not None)
 if opts.output_filename.endswith(('.xml', '.xml.gz')):
-    process = ligolw_process.register_to_xmldoc(xmldoc, "sbank",
-        opts_dict, version="no version",
+    process = xmldoc.register_process(
```

Comment:
given this change, it looks like we can remove the ligolw_process import

Reply:
Done